lib/ui/text/paragraph.cc

```diff
@@ -40,156 +42,67 @@ IMPLEMENT_WRAPPERTYPEINFO(ui, Paragraph);
 
 DART_BIND_ALL(Paragraph, FOR_EACH_BINDING)
 
-Paragraph::Paragraph(PassOwnPtr<RenderView> renderView)
-    : m_renderView(renderView) {}
+Paragraph::Paragraph(PassOwnPtr<RenderView> renderView) {
+  m_paragraphImpl = std::make_unique<ParagraphImplBlink>(renderView);
+}
+
+Paragraph::Paragraph(std::unique_ptr<txt::Paragraph>* paragraph) {
```

Comment:
This should be a std::unique_ptr<txt::Paragraph> not a std::unique_ptr<txt::Paragraph>*